core/src/main/scala/sttp/tapir/Codec.scala

```diff
@@ -91,7 +93,50 @@ object Codec extends MultipartCodecDerivation with FormCodecDerivation {
 
   def stringCodec(charset: Charset): PlainCodec[String] = plainCodec(identity, charset)
 
-  private def plainCodec[T: Schema](parse: String => T, charset: Charset = StandardCharsets.UTF_8): PlainCodec[T] =
+  implicit val localTimePlainCodec: PlainCodec[LocalTime] = plainCodec[LocalTime](LocalTime.parse)
+  implicit val localDatePlainCodec: PlainCodec[LocalDate] = plainCodec[LocalDate](LocalDate.parse)
+
+  implicit val zonedDateTimePlainCodec: PlainCodec[ZonedDateTime] =
```

Comment:
there are also schemas for: `OffsetDateTime`, `Date`, `ZoneOffset`, `Duration` (java & scala), `OffsetTime`. Maybe we should add codecs for these as well, so that there's codecs for everything that has a schema?